Gemfile

```diff
@@ -7,3 +7,34 @@ gem "pg", "~> 1.0", platform: :ruby
 gem "activerecord-jdbcpostgresql-adapter", platform: :jruby
 gem "ffi-geos", platform: :jruby
 gem "byebug" if ENV["BYEBUG"]
+
+def activerecord_version
```

Comment:
This gets only one version, shouldn't we test against multiple rails versions ? For that, we could do as with ruby: take advantage of the test matrix in github ci.

In that case the version could simply be an env variable or a task argument

Reply:
I'm not certain why having both is mandatory. Actually I think I am missing the point of this method :/

Reply:
Gotcha. The point of this method is that for local development if we want to test against ActiveRecord, we need to add Rails to the bundle and fetch it from its git repo in order to access all of the tests. If we just use the ActiveRecord from Rubygems the test directory is not included, same with Rails from Rubygems.

So this method reads in our gemspec, finds what ActiveRecord version we've specified as a dependency, fetches all of the Rails releases from Rubygems, finds the highest matching version (i.e. ActiveRecord ~> 7.0.0 resolves to ActiveRecord=7.0.2.3 currently), and then pulls the corresponding Rails version from git since all releases are tagged on git as well.